src/languages/en-US/arguments.json

```
@@ -9,6 +9,7 @@
 	"categoryChannel": "I could not resolve `{{parameter}}` to a category channel, please make sure you typed its name or ID correctly!",
 	"channel": "I could not resolve `{{parameter}}` to a channel, make sure you typed its name or ID correctly!",
 	"command": "I could not resolve `{{parameter}}` to a command! Make sure you typed its name or one of its aliases correctly!",
+	"commandMatch": "I could not resolve `{{parameter}}` to a command nor a category match, there are some of the valid formats:\n\n- `ping` (matches the ping command only).\n- `social.*` (matches all commands under the **Social** category).\n- `social.member management.*` (matches all commands under **Social** > **Member Management** sub-category).\n\n> **Hint**: When checking `{{commandContext.commandPrefix}}help`, the category of each command is located at the top of the embed.",
```

```suggestion
	"commandMatch": "I could not match `{{parameter}}` to a command nor category, these are some of the valid formats:\n\n- `ping` (matches the ping command only).\n- `social.*` (matches all commands under the **Social** category).\n- `social.member management.*` (matches all commands under **Social** > **Member Management** sub-category).\n\n> **Hint**: When checking `{{commandContext.commandPrefix}}help`, the category of each command is located at the top of the embed.",
```

Grammar nazi in da house